app/common-messages.js

```diff
@@ -33,6 +33,54 @@ export default defineMessages({
   achievements: {
     id: 'app.common.achievements',
   },
+  moderation: {
```

Comment:
common-messages.js should be used only for messages that are used in more than one component / container. These messages should moved to local messages in the new component.